src/routes/auth/controllers.js

```diff
@@ -3,7 +3,7 @@ export const login = async (req, res) => {
  
   try {
     let myPost = req.body;
-    let authenticated =  users.find(e => e.userName.toLowerCase().includes(myPost.userName));
+    let authenticated =  users.find(e => e.userName === myPost.userName);
```

Comment:
What is myPost stand for?

Reply:
myPost stands for the POST request coming in, Maybe I should have chosen a better more meaningful name

Reply:
renamed myPost to postBody

Reply:
Hi Sadat, as much as I remember you had already revised this, hadn't you?